<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VectorWin</class>
 <widget class="QWidget" name="VectorWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>向量一</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>72</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>向量二</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>计算选项</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>求模</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>内积</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>结果</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_info">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>输入用英文逗号隔开的数字来代表向量</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>111</width>
     <height>65</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>重置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Go">
      <property name="text">
       <string>计算</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>textBrowser</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
